Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Using the official python 2.7 as a Base Image
+FROM python:2
+
+ADD . /edx-e2e-tests
+WORKDIR /edx-e2e-tests
+
+# Configuration
+RUN apt-get update
+RUN pip install paver
```

Comment:
With the exception of the virtualenv, I think all pip installs should be coming from requirements. This way, there can be no difference between the versions run by Jenkins and locally